### ISSUE TITLE
Open recently closed tab in same tab

### ISSUE THREAD

**Issue by orschiro** [quodroc/HumbleNewTabPage#29]
Dear @quodroc,

Currently recently closed tabs open in a new tab, meaning that I am left with an unused humble new tab. Could this behaviour be changed to open the recently closed tab in the same tab (the current humble new tab page). 

Thankfully,

Robert

**Comment by quodroc** [comment]
This is a bit tricky. I'm using chrome.sessions.restore which preserves the tab's history, scroll position, etc. but doesn't have the option to reuse the current tab. What I could do is close the new tab page when you restore a closed tab although it's not a perfect solution.

**Comment by orschiro** [comment]
> but doesn't have the option to use the current tab

Is it worth filing a feature request for it in the Chromium bug tracker?

> What I could do is close the new tab page when you restore a closed tab although it's not a perfect solution.

That would be great for a temporary workaround!

**Comment by quodroc** [comment]
> Is it worth filing a feature request for it in the Chromium bug tracker?
Maybe but I doubt they would add it. It's a pretty specific use case and it's not clear what they would do about the history of the two tabs (combine them?).

**Comment by orschiro** [comment]
> Maybe but I doubt they would add it. It's a pretty specific use case and it's not clear what they would do about the history of the two tabs (combine them?).

Good point...Then I would very welcome the workaround if you find time to implement it. 

Thanks,

Robert
